cypress/support/pages/settings.password.pageObject.js

```diff
@@ -0,0 +1,9 @@
+class SettingsPassword {
+  updatePassword(newPassword) {
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('[placeholder="Password"]').clear().type(newPassword);
```

Comment:
Consider using `data-cy` attributes for selecting elements in your tests as per the task requirements. This will make your tests more robust and less prone to breaking if the UI changes.